docs/core/runtime-config/debugging-profiling.md

```diff
@@ -36,19 +38,31 @@ This article details the settings you can use to configure .NET debugging and pr
 | | Setting name | Values |
 | - | - | - |
 | **runtimeconfig.json** | N/A | N/A |
-| **Environment variable** | `CORECLR_PROFILER` | *string-guid* |
+| **Environment variable** | `CORECLR_PROFILER` or `DOTNET_PROFILER` | *string-guid* |
 
 ## Profiler location
 
+Once profiling is enabled, the profiler can be loaded in two ways, with environment variables (cross-plat) or through the registry (Windows only).
+
+### Environment Variable (cross-plat)
+
 - Specifies the path to the profiler DLL to load into the currently running process (or 32-bit or 64-bit process).
 - If more than one variable is set, the bitness-specific variables take precedence. They specify which bitness of profiler to load.
-- For more information, see [Finding the profiler library](https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/profiling/Profiler%20Loading.md).
+- If any of these environment variables are present, the registry look up is skipped altogether, and the path from `DOTNET_PROFILER_PATH` is used to load the DLL.
```

Comment:
```suggestion
```
It might be any of the environment variables that take precedence, not solely DOTNET_PROFILER_PATH. I suggested a different way we could phrase it just above the environment variable section.